src/sdk/sql_cluster_router.cc

```diff
@@ -1761,23 +1761,24 @@ base::Status SQLClusterRouter::HandleSQLCreateTable(hybridse::node::CreatePlanNo
     ::openmldb::nameserver::TableInfo table_info;
     table_info.set_db(db_name);
 
-    if (!cluster_sdk_->IsClusterMode()) {
-        if (create_node->GetReplicaNum() != 1) {
-            return base::Status(base::ReturnCode::kSQLCmdRunError,
-                                "Fail to create table with the replica configuration in standalone mode");
-        }
-        if (!create_node->GetDistributionList().empty()) {
-            return base::Status(base::ReturnCode::kSQLCmdRunError,
-                                "Fail to create table with the distribution configuration in standalone mode");
-        }
+    auto ns_client = cluster_sdk_->GetNsClient();
+    std::vector<::openmldb::client::TabletInfo> tablets;
+    std::string msg;
+    bool ok = ns_client->ShowTablet(tablets, msg);
+    if (!ok) {
+        return base::Status(base::ReturnCode::kCreateTableFailedOnTablet, "set default replica num failed");
     }
+    int tablets_size = tablets.size();
```

Comment:
can get all tablets by `cluster_sdk_->GetTablet()`

Reply:
cluster_sdk_->GetTablet() must pass `table_name` as parameter, it seems have no method to get all tablets in cluster
<img width="784" alt="image" src="https://user-images.githubusercontent.com/60570208/163101520-7655a74b-b4eb-47e8-a4cf-1fc1d7eef25d.png">

Reply:
Add `GetAllTablet` method to `ClientManager`.